src/content/docs/new-relic-solutions/increase-user-engagement.mdx

```diff
@@ -0,0 +1,196 @@
+
+<img
+    title="Adjust timeslice data"
+    alt="Adjust timeslice data"
+    src={timeWPM}
+/>
+
+New Relic contextualizes this data over time. You can click and drag over relevant parts of this chart, which gives you more granular insight about site behavior. When you're finished, click the time stamp button at the top to adjust back to your preferred time period. 
+
+    </Collapser>
+    <Collapser
+        id="content"
+        title="Compare content size, load times, and sources"
+    >
+<img
+    title="Analyze content size"
+    alt="Analyze content size"
+    src={contentWPM}
+/>
+
+WPM lets you visualize how different types of assets perform relative to each other. In the **Core web vitals** section, you can view four bar graphs that show your content's size, load time, their resources, and how the network affects them. Using two or more of these graphs can help you identify if, say, your CSS assets are causing load issues. 
+
+If page load speed is slow, then you can use the **Content** section to diagnose potential sources of the problem relative to each other. For example, comparing **Content size** to **Content load time** can give insight into whether CSS is taking up too many resources and loading last, which would cause a delay in largest contentful paint. 
+
+    </Collapser>
+    <Collapser
+        id="geo"
+        title="Fix broken links"
+    >
+
+Unlike a website grader, WPM lets you view your critical pages in context of each other. If user engagement is down but your site is available and content is optimized, then something else might be at play. With WPM, your monitors routinely ping your chosen pages and report back. A possible journey to troubleshoot might look like:
+
+1. Go to **[one.newrelic.com > Website performance monitoring](https://one.newrelic.com/website-grader)**.
+2. You check the availability of your page. It reads as available. 
+3. Your core web vitals are reporting as **Good** across the board.
+4. You notice the timeslice data shows a steep drop-off in traffic the day before, around the time updates to your landing page were made.
+5. On the WPM page for your landing page, you see a broken link has been reported. It's a boilerplate banner that's used across other areas of your site.
+
+At this point, you can coordinate with the relevant teams to fix the broken link and deploy an update for the rest of the site. 
+
+    </Collapser>
+</CollapserGroup>
+    </Step>
+</Steps>
+
+## What's next? [#next]
+
+WPM is designed to simplify getting detailed insight about your site. But you can use New Relic to collect data about real user traffic or key user flows. Depending on your organization's structure, you might need to loop in your developer team to set these up. We recommend you start with these two features: 
```

Comment:
I’m confused by “these two features:” but there are three bullet points that follow.

Reply:
LOL good catch! ty